src/diff_opt.jl

```diff
@@ -296,10 +310,37 @@ function MOI.set(model::Optimizer, ::BackwardInVariablePrimal, vi::VI, val)
     return
 end
 
-function MOI.get(model::Optimizer, ::BackwardOut{LinearObjective}, vi::VI)
-    return _get_dc(model, vi)
+function MOI.get(model::Optimizer, ::BackwardOutObjective)
```

Comment:
You rarely want all the sensitivities.
If you have a model with many variables this will waste a lot of time while the user might just care about a single variable coefficient.